### DISCUSSION TITLE
Imput and Output

### DISCUSSION THREAD

**Discussion by Ll1nusllove** [b-inary/postflop-solver#22] [General]
Hi, I know that currently you are working on the export and import of calcs, but I am just interested in how much will it approximately take, since you said before, that it can take about 2 months, so can we expect this feature in next 1-2 weeks, or it will take way more time?
Btw, thank you so much for this project. You are battling alone with whole companies and making even a better engine for free, than they are for tons of money... 
It's so cool...

**Comment by b-inary** [comment]
Sorry for the delay in implementing the save/load feature. The file I/O logics are almost done, but there is still an additional task to integrate the result viewer with the just-in-time solving feature, which is required when loading a game tree without post-river information. The release date depends heavily on my motivation and available schedule; if I concentrate on it, I think it will only take about a week, but in practice I don't think it will go that way. However, I would like to release the new version by the end of April at the latest.
